HPXMLtoOpenStudio/measure.rb

```diff
@@ -4169,15 +4191,102 @@ def self.set_surface_exterior(model, spaces, surface, exterior_adjacent_to)
     elsif [HPXML::LocationBasementConditioned].include? exterior_adjacent_to
       surface.createAdjacentSurface(create_or_get_space(model, spaces, HPXML::LocationLivingSpace))
       @cond_bsmnt_surfaces << surface
+    elsif [HPXML::LocationOtherHeatedSpace, HPXML::LocationOtherMultifamilyBufferSpace, HPXML::LocationOtherNonFreezingSpace].include? exterior_adjacent_to
+      add_otherside_coefficients(surface, exterior_adjacent_to, model, spaces)
+      surface.setSunExposure('NoSun')
+      surface.setWindExposure('NoWind')
     else
       surface.createAdjacentSurface(create_or_get_space(model, spaces, exterior_adjacent_to))
     end
   end
 
+  def self.add_otherside_coefficients(surface, exterior_adjacent_to, model, spaces)
+    if spaces[exterior_adjacent_to].nil?
+      # Create E+ other side coefficient object
+      otherside_object = OpenStudio::Model::SurfacePropertyOtherSideCoefficients.new(model)
+      otherside_object.setName(exterior_adjacent_to)
+      # Fixme: assumption the same as SurfacePropertyConvectionCoefficients of return air plenum
+      otherside_object.setCombinedConvectiveRadiativeFilmCoefficient(30)
+      # Schedule of space temperature, can be shared with water heater/appliances
+      create_outside_boundary_schedule(model, exterior_adjacent_to, spaces)
+      # FIXME: wait for new OS release with bugfix of https://github.com/NREL/OpenStudio/issues/3848
+      otherside_object.setPointer(9, @mf_temp_sch_map[exterior_adjacent_to].handle)
```

Comment:
It doesn't seem like we actually need this `@mf_temp_sch_map` global variable. Can't we obtain the schedule from the `SurfacePropertyOtherSideCoefficients` object, which itself can be retrieved from the `spaces` variable if needed?